app/helpers/template_helpers.py

```diff
@@ -217,31 +240,55 @@ def get_static_content_urls(language_code: str, base_url: str, schema_theme: str
     if language_code == "cy":
         help_path = "help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein/"
         cookies_path = "cwcis/"
-        accessibility_statement_path = "hygyrchedd/"
+        accessibility_statement_path = "datganiad-hygyrchedd/"
         privacy_and_data_protection_path = "preifatrwydd-a-diogelu-data/"
         terms_and_conditions_path = "telerau-ac-amodau/"
         contact_us = "cysylltu-a-ni/"
+        languages = "help/ieithoedd-a-hygyrchedd/ieithoedd/"
+        bsl_and_audio_videos = (
+            "help/ieithoedd-a-hygyrchedd/hygyrchedd/fideos-hygyrch-gyda-bsl/"
+        )
 
-    else:
+    elif schema_theme == "census-nisra":
         help_path = (
             "help/help-with-the-questions/online-questions-help/"
             if schema_theme == "census-nisra"
             else "help/how-to-answer-questions/online-questions-help/"
         )
         cookies_path = "cookies/"
-        accessibility_statement_path = "accessibility/"
+        accessibility_statement_path = "accessibility-statement/"
         privacy_and_data_protection_path = "privacy-and-data-protection/"
         terms_and_conditions_path = "terms-and-conditions/"
         contact_us = "contact-us/"
 
-    return {
+    else:
+        help_path = (
+            "help/help-with-the-questions/online-questions-help/"
+            if schema_theme == "census-nisra"
+            else "help/how-to-answer-questions/online-questions-help/"
+        )
```

Comment:
We wouldn't get to this else block if `schema_theme == "census-nisra"` due to the `elif` above so this logic can be removed.

Reply:
Some of the remnants of the old logic I forgot to look at, fixed, thanks!